chromium/browser_type_test.go

```diff
@@ -150,3 +153,87 @@ func TestBrowserTypePrepareFlags(t *testing.T) {
 		})
 	}
 }
+
+func TestExecutablePath(t *testing.T) {
```

Comment:
This test doesn't test the `ErrChromeNotInstalled`, but at least, it verifies the `ExecutablePath`'s behavior.

Reply:
Why doesn't it test for that?

Reply:
I think if you follow your [comment](https://github.com/grafana/xk6-browser/pull/1137/files#r1425442535) then that error could be returned from `ExecutablePath` instead of a `bool`, which would allow a test for that error in here.

Reply:
See 5499e10.